core/src/main/kotlin/org/evomaster/core/search/service/Statistics.kt

```diff
@@ -256,4 +260,29 @@ class Statistics : SearchListener {
                             .count()
                 }
     }
+
+    fun writeCoveredTargets(solution: Solution<*>, format : EMConfig.SortCoveredTargetBy){
```

Comment:
where is the option to write a simple txt file with just a single column including only the unique target ids?